config/locales/en.yml

```diff
@@ -4956,6 +4958,10 @@ en:
   repair_external_photos: Repair External Photos
   replace_boundary_with_kml: Replace boundary with KML
   replace_parentesis: (replace)
+  # Label for a button that confirms the replacement of the userName
+  replace_user_name_admin_confirmation: Yes, replace
+  # Message displayed in the modal of replacement of the userName as explanation of the action
+  replace_user_name_admin_explanation: Are you sure you want to replace this user's username with an anonymized one? This cannot be undone. The new username will look like
```

Comment:
Again, we don't want this text to be localized, but if we did, we want to avoid sentence fragments like "The new username will look like". To give translators more control over ordering you would use a variable if the value was genuinely dynamic, but here you would just need to make a translatable string with some HTML.